### DISCUSSION TITLE
[Question] Calendar component vs mdehoog calendar.

### DISCUSSION THREAD

**Discussion by hakeem86** [fomantic/Fomantic-UI#249] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
Hi,

Today I'm using semantic-ui with [mdehoog's calendar component](https://github.com/mdehoog/Semantic-UI-Calendar).
I'm planing to move to fomantic-ui, but it has the calendar module integrated to fomantic-ui. 

Is the calendar module the exactly same? just forked into fomantic-ui? or is there any differences in the api?

**Comment by y0hami** [comment]
The calendar component in FUI is the exact same as mdehoog's repo. In fact, we merged the repo into FUI.

You can see the issue [here](https://github.com/fomantic/Fomantic-UI/issues/8) where you can find some more information.

**Comment by lubber-de** [comment, marked as answer]
Infact, we already enhanced mdehoog's version but remain a backward compatible API. So your existing code should work just fine with FUI with the benefit of gaining more features (especially by the fresh 2.6.4 release) like [disabledDaysOfWeek](https://github.com/fomantic/Fomantic-UI/pull/199), [disabledDates](https://github.com/fomantic/Fomantic-UI/pull/211),[ showWeekNumbers](https://github.com/fomantic/Fomantic-UI/pull/113), [min/maxDate in MetaData](https://github.com/fomantic/Fomantic-UI/pull/177) and more improvements already in [progress](https://github.com/fomantic/Fomantic-UI/pull/198) :grin: 
More information at https://fomantic-ui.com/modules/calendar.html#/settings

**Comment by hakeem86** [comment]
Complete forgot to say thank you.. thanks for the answer :)
